### ISSUE TITLE
Broken runtime_data directory reference

### ISSUE THREAD

**Issue by lecifire** [connervieira/Phantom#1]
Followed the installation steps without any issue however encountered this error when trying to run alpr:

$ alpr test.jpg

{"error": "The run-time directory '/usr/share/openalpr/runtime_data' does not exist."}
{"error": "Error loading Phantom"}

Running $ alpr --version is able to provide me this:

alpr  version: 1.0.0

**Comment by connervieira** [comment]
That's pretty strange. I previously had issues where the opposite problem would occur, such that Phantom would create a directory titled "openalpr" when "phantom" was the intended name.

I figure there's a preset somewhere that's set incorrectly. I'll take a look and update the issue.

Just to clarify, are you using the latest version of Phantom from the git repository?

**Comment by connervieira** [comment]
I was able to recreate the issue in a virtual machine, and I think I found a solution. I tested in a clean install of Pop!_OS 22.04, and it looks like everything is working now.

Try fetching the latest changes from the git repository, and re-compiling/installing. You might need to delete old support files before running `sudo make install`.

This should remove all support files, but will also clear any configuration changes you've made: `sudo rm -rf /etc/openalpr/ /etc/phantom/ /usr/share/openalpr /usr/share/phantom`

I'm tentatively closing this issue, but I will re-open it if the issue persists.
